packages/components/src/components/modal/modal.scss

```diff
@@ -89,11 +89,13 @@
     }
     /** @define dc-modal-header */
     &-header {
+        align-items: center;
+        border-bottom: 2px solid var(--general-section-1);
```

Comment:
@artsemkavaleu-deriv  Please don't change the components package code, that's reusable code.

Reply:
sure, thx for notice!